### DISCUSSION TITLE
Example of how to retrieve active input?

### DISCUSSION THREAD

**Discussion by Jibun-no-Kage** [WesSouza/lgtv-ip-control#109] [Q&A] [answered]
Example of how to retrieve active input?

**Comment by WesSouza** [comment, marked as answer]
Apparently on LG TVs everything is considered an app, so to get the current input, you call `getCurrentApp`:

```js
const { LGTV } = require("lgtv-ip-control");

const tv = new LGTV("IP", "MAC", "CODE");
tv.connect()
  .then(async () => {
    console.log(await tv.getCurrentApp());
    tv.disconnect();
  })
  .catch((e) => {
    console.error(e);
    tv.disconnect();
  });
```

Running this example on my LG TV set to HDMI 1 resulted in the following output:
```
➜ node test.js
com.webos.app.hdmi1
```

**Comment by Jibun-no-Kage** [comment]
Actually I figured this out... and forgot to close the question... but just the same thank you!  Really appreciate you taking the time to respond!
